relayer/naive-strategy.go

```diff
@@ -563,7 +562,7 @@ func (nrs *NaiveStrategy) RelayPackets(src, dst *Chain, sp *RelaySequences) erro
 	}
 
 	if len(msgs.Src) != 0 {
-		updateMsg, err := dst.UpdateClient(src)
+		updateMsg, err := src.UpdateClient(dst)
```

Comment:
Was this wrong before?

Reply:
yes pretty sure. The logic doesn't hold since if you look at the code, this message is beings added to msgs.Src